python/ray/autoscaler/autoscaler.py

```diff
@@ -77,6 +77,11 @@
             "worker_ips": (list, OPTIONAL),  # local cluster worker nodes
             "use_internal_ips": (bool, OPTIONAL),  # don't require public ips
             "extra_config": (dict, OPTIONAL),  # provider-specific config
+
+            # Whether to try to reuse previously stopped nodes instead of
+            # launching nodes. This will also cause the autoscaler to stop
+            # nodes instead of terminating them. Only implemented for AWS.
+            "cache_stopped_nodes": (bool, OPTIONAL),
```

Comment:
Why make this part of the config rather than the `ray down` command?

Reply:
This doesn't just apply to down, but to autoscaling as a whole.